Tests/ApolloCodegenTests/CodeGenIR/IRInputObjectTests.swift

```diff
@@ -25,7 +25,7 @@ class IRInputObjectTests: XCTestCase {
 
   func buildSubject() async throws {
     let ir: IRBuilder = try await .mock(schema: schemaSDL, document: document)
-    subject = ir.schema.referencedTypes.inputObjects.first!
+    subject = ir.schema.referencedTypes.inputObjects[1]
```

Comment:
Why would we change this to the second object of the array instead of the first?

Reply:
Because when the types are sorted, `Input` is the second type in referencedTypes, instead of first.